distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -580,9 +580,9 @@ type File
         if Context.Output.is_enabled.not then Error.throw (Forbidden_Operation.Error "File moving is forbidden as the Output context is disabled.") else
             File_Error.handle_java_exceptions self <| case replace_existing of
                 True ->
-                  copy_options = [StandardCopyOption.REPLACE_EXISTING].to_array
-                  self.move_builtin destination copy_options
-                False -> self.move_builtin destination Array.empty
+                  copy_options = [StandardCopyOption.REPLACE_EXISTING]
+                  self.move_builtin destination copy_options.to_array
```

Comment:
Nobody shall require `.to_array` call. The `File` should be fixed to work with `Vector` or any array-like object. The issue is already tracked as 
- #7133